bindings/Python/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ Python_add_library(adios2_py MODULE
   py11Operator.cpp
   py11File.cpp py11File.tcc
   py11glue.cpp
+  WITH_SOABI
```

Comment:
According to the documentation, the argument needs to directly follow the library type:
```
Python_add_library (<name> [STATIC | SHARED | MODULE [WITH_SOABI]]
                    <source1> [<source2> ...])
```